open-metadata-implementation/framework-services/ocf-metadata-management/ocf-metadata-spring/src/main/java/org/odpi/openmetadata/frameworkservices/ocf/metadatamanagement/server/spring/ConnectedAssetResource.java

```diff
@@ -163,6 +163,30 @@ public AssetResponse getConnectedAssetSummary(@PathVariable String   serverName,
     }
 
 
+    /**
+     * Returns the anchor asset of an entity.
+     *
+     * @param serverName  String   name of server instance to call.
```

Comment:
the presence of String is inconsistent with the javadoc of the rest of the module. some for other parameters

Reply:
I've seen this format used for other endpoints underneath this new one, such as "getAssetSummary", "getCertifications" etc. Are you sure I should remove it?
This also applies for the 'or' at the end of the lines for the exceptions.

Reply:
yes you are right - sorry about that